packages/Confirmation/stories/Confirmation.stories.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import { withKnobs } from "@storybook/addon-knobs";
+import L10n from "@paprika/l10n";
+import ShowcaseStory from "./examples/Showcase";
+import ConfirmationExample from "./examples/ConfirmationExample";
+
+storiesOf("Confirmation", module)
+  .addDecorator(withKnobs)
+  .add("Showcase", ShowcaseStory)
+  .add("Confirmation", () => <ConfirmationExample />)
+  .add("ConfirmationExample with locale", () => (
```

Comment:
It would be nice to build a story that supports the use case that drove this task:
https://github.com/acl-services/paprika/issues/156